.rubocop.yml

```diff
@@ -86,7 +86,7 @@ Style/Next:
   Enabled: false
 
 Metrics/AbcSize:
-  Max: 30
+  Enabled: false
```

Comment:
Inspired by https://github.com/theforeman/smart-proxy/commit/21642e60255d5ad19e30a97e706ffc7afb6defa4, I don't remember last time the AbcSize metric was of any use for me.